qcelemental/models/v1/procedures.py

```diff
@@ -332,8 +352,18 @@ def convert_v(
         if check_convertible_version(version, error="TorsionDriveResult") == "self":
             return self
 
+        opthist_class = next(iter(self.optimization_history.values()))[0].__class__
         dself = self.dict()
         if version == 2:
+            # remove harmless empty error field that v2 won't accept. if populated, pydantic will catch it.
+            if dself.pop("error", None):
```

Comment:
Consider removing the if statement (since it doesn't do anything) and change to `dself.pop("error", None)`.

Reply:
fixed here and two other spots in next PR